### PR TITLE
[RFC] Fix the update URIs

### LINKED ISSUE #60: TabNine Installation Script Updated
https://github.com/codota/TabNine/blob/master/dl_binaries.sh

- The URL to get the new version has changed.
  - `https://update.tabnine.com/version` -> `https://update.tabnine.com/bundles/version`
- The URL to download binaries has changed.
  - `https://update.tabnine.com/$path` -> `https://update.tabnine.com/bundles/$path/TabNine.zip`

### PR DESCRIPTION
**Pull request by Shougo**
Fix #60

### CODE REVIEW

**Comment by Shougo** [comment]
Powershell maintainer is needed...

**Comment by Shougo** [comment]
Windows testers are needed.

File: install.sh [hunk 1]
```diff
@@ -15,11 +15,14 @@ esac
...
     if [ -f binaries/$path ]; then
         exit
     fi
     echo Downloading version $version $target
-    curl https://update.tabnine.com/$path --create-dirs -o binaries/$path
-    chmod +x binaries/$path
+    curl https://update.tabnine.com/bundles/$zip --create-dirs -o binaries/$zip
+    cd $(dirname binaries/$zip)
+    unzip -o TabNine.zip
+    chmod +x TabNine
```
Comment (dimacodota) [inline install.sh]: @Shougo - you need to `chmod` all the executables  in the directory

Comment (dimacodota) [reply install.sh]: WD-TabNine
TabNine
TabNine-deep-local
TabNine-deep-cloud

Comment (dimacodota) [reply install.sh]: and delete the zip file

Comment (Shougo) [reply install.sh]: Thanks.  Fixed.

**Comment by ress997** [comment]
The directory structure is wrong.
image

**Comment by Shougo** [comment]
Fixed.

**Comment by ress997** [comment]
Tried the newer version on the M1 Mac and confirmed that the problem was fixed.

**Comment by nekowasabi** [comment]
Worked by Windows gvim and Macbook Pro(2020).

**Comment by Shougo** [comment]
Thanks!

**Comment by Shougo** [comment]
@tbodt Can you merge it?  The current installation script is not compatible with the latest.

**Comment by tbodt** [comment]
Why would they change the download process... Thanks for fixing it.

**Comment by Shougo** [comment]
Thanks!

> Why would they change the download process... Thanks for fixing it.

They split the binary...

**Pull request merged**